changelogs/fragments/355_l3_interace_fix.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - fixes https://github.com/ansible-collections/cisco.ios/issues/350 l3_interfaces module and Unit Test cases added
```

Comment:
Can we have some more details on the actual changes in this PR or what it fixes? The URL to the issue can supplement it. Also, these are rendered in RST, so we can have proper anchors like [here](https://github.com/ansible-collections/ansible.netcommon/pull/292/files#diff-bc3c8f8cb6bf7f28ecd8909343dc7c9c8d8ebaa8e2eaaae4eab98e45ea54ef98R7).